paddle/phi/kernels/cpu/conv_grad_kernel.cc

```diff
@@ -101,3 +131,14 @@ PD_REGISTER_KERNEL(depthwise_conv2d_grad,
 
 PD_REGISTER_KERNEL(
     conv3d_grad, CPU, ALL_LAYOUT, phi::Conv3DGradKernel, float, double) {}
+
+PD_REGISTER_KERNEL(
+    conv2d_grad_grad, CPU, ALL_LAYOUT, phi::ConvGradGradKernel, float, double) {
```

Comment:
conv2d_grad_grad -> cond2d_double_grad?

Reply:
All naming of conv double grad kernel is invalid, include conv3d, normlize them in another PR